utils/dbutils.ts

```diff
@@ -1,92 +0,0 @@
-import sqlite from 'better-sqlite3';
```

Comment:
No need to keep this outside of `src/` really - it sort of belongs there even, I'd say